frontend/src/components/placementVisuals/Placement.tsx

```diff
@@ -31,6 +37,57 @@ function Placement({ stepID, sampleIDs }: PlacementProps) {
     const destinationContainers = labworkStepPlacement.destinationContainers
     const activeSourceContainer = labworkStepPlacement.activeSourceContainer
     const activeDestinationContainer = labworkStepPlacement.activeDestinationContainer
+    const step = selectStepsByID(store.getState())[stepID]
+    const usesSamplesheet = step.name === EXPERIMENT_RUN_ILLUMINA_STEP
+    const cells = useAppSelector((state) =>  activeDestinationContainer?.name && selectContainer(state)({ name: activeDestinationContainer.name })?.cells)
+
+    const handleGetSamplesheet = useCallback(async () => {
```

Comment:
I feel like this function belongs in `frontend/src/modules/labworkSteps/actions.ts`

Reply:
I do disagree, but i'll move it ...